docs/diagrams/UiSequenceDiagram.puml

```diff
@@ -1,4 +1,6 @@
 @startuml
+!include docs/diagrams/Style.puml
+
```

Comment:
I think the arrow when a method returns should be dashed, Try using "-->" instead of "->".